content/docs/started/cloud/getting-started-icp.md

```diff
@@ -192,3 +192,32 @@ kfctl delete all --delete_storage
 # from mlpipeline.
 kfctl delete all
 ```
+## TroubleShooting
+### Insufficient Pods 
+  
+Default installation of IBM Cloud Private(ICP) configures 10 allocatable pods per core. So if you have 8 cores, there will be 80 allocatable pods. However, execution of ICP (from default installation) requires ~40 pods and Kubeflow requires ~45 pods, so you may have insufficient pods issue.  
+
+To increase the number of pods, you can edit /etc/cfc/kubelet/kubelet-service-config, increase podsPerCore from 10 to 20 (or 30). Save your change and restart kubelet.
+```
+systemctl restart kubelet
+```
+
+### Kubeflow Central Dashboard: connection reset error or slow response   
+This may happen when there is not enough memory (e.g. if you only have 16G memory). You may consider turning off some non-essential services in ICP
```

Comment:
Please also update all `ICP` to IBM Cloud Private in the description (if in the coding or CLI outputs, we can keep that). Thanks!

Reply:
@jinchihe Thanks again. I have modified the file according to your feedback. Please have a look and let me know if further change is needed.

Reply:
/assign @jinchihe